AEM/optimum-habana-perf.md

```diff
@@ -77,14 +80,14 @@ QUANT_CONFIG=./quantization_config/maxabs_measure.json python ../gaudi_spawn.py
 --batch_size 1
 ```
 
-## Llama3
+### Llama3
 Here is an example to measure the tensor quantization statistics on Llama3 with 8 cards:
-> Please note that Llama3-405B requires minimum 8 cards Gaudi3.
+> Please note that Llama3-405B requires minimum 8 Gaudi3 cards.
```

Comment:
Please note that Llama3-405B requires **a** minimum 8 Gaudi3 cards.

Reply:
addressed it accordingly.